### ISSUE TITLE
Admin can edit list of Analysts

### ISSUE THREAD

**Issue by MarsRomer2** [bcgov/CONN-CCBC-portal#1039]

**Comment by AntBush** [comment]
Discussion from backlog refinement: 
- Include an active column in the analyst table
- The delete function will be permanent, while the active toggle is a "hide" if active is set to false (probably needs to update the RLS)
- History should be tracked in the new history table (from #1208 that is an offshoot of #801)

**Comment by MarsRomer** [comment]
@elliottSlee  I have updated this ticket to remove the Edit and Delete functionality which is now in ticket #1250.  Would you be able to update the screen designs in this ticket please?

**Comment by elliottSlee** [comment]
@MarsRomer updated & ready to go

**Comment by MarsRomer** [comment]
@marcellmueller 
Love the UI!  
I did notice some unexpected behavior with these steps:
1. Add new name (Mars)
2. Assign Lead = Mars on CCBC-010002 on the dashboard
3. De-activate Mars on Admin tab
4. Return to dashboard to see that Lead has now changed to unassigned. (error 1)
5. Return to admin tab and activate Mars
6. On Dashboard Mars reappears in Lead for that application
7. Return to list of analysts and Mars is deactivated (error 2)

I was able to recreate this with another name in the list (Mars 2).

**Comment by marcellmueller** [comment]
@MarsRomer I don't think that is a bug related to this ticket. Though if you want non active analyst to show up in the dropdown still if they are assigned we can fix that.

**Comment by MarsRomer** [comment]
I don't expect it to show in the dropdown if they have been deactivated.  But if someone was assigned as Lead and then de-activated, it should still show them as the Lead.  This is what is not happening

**Comment by marcellmueller** [comment]
> I don't expect it to show in the dropdown if they have been deactivated. But if someone was assigned as Lead and then de-activated, it should still show them as the Lead. This is what is not happening

Yeah that's fair, I was just saying that isn't part of this ticket which is to list, add and change active status.

**Comment by MarsRomer** [comment]
oh I was sure we had talked about it couple of times and put that in the ticket, I'm not sure how that was missed.  I am pondering next steps now, I don't want to push this to prod the way it behaves....

**Comment by marcellmueller** [comment]
> oh I was sure we had talked about it couple of times and put that in the ticket, I'm not sure how that was missed. I am pondering next steps now, I don't want to push this to prod the way it behaves....

This is a bug in the assign lead dropdown,  not this ticket though you noticed it because of being able to deactivate analysts.

I can work on it (it should be a bug ticket), shouldn’t be too difficult to fix. Unless you are going to deactivate someone immediately I don’t know that it has to block pushing to prod. 

The analyst is still assigned in the database either way, the dropdown only queries active analysts so it just doesn’t show their name.

**Comment by MarsRomer** [comment]
what did you make of the other error? 
"7. Return to list of analysts and Mars is deactivated (error 2)"

**Comment by marcellmueller** [comment]
> what did you make of the other error?
> "7. Return to list of analysts and Mars is deactivated (error 2)"

Oh I missed that sorry. I figured out the cause of it and have a fix incoming.

**Comment by MarsRomer** [comment]
ku ku ku
